src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const newPeople = people.map((person: Person) => {
+    const newPerson: Person = { ...person };
+
+    newPerson.mother = people.find((el) => el.name === person.motherName);
+    newPerson.father = people.find((el) => el.name === person.fatherName);
+
+    return newPerson;
+  });
+
+  if (!people.length) {
+    return null;
```

Comment:
The check for 'people.length' is done after the loop in line 10. Consider moving this up before the loop to avoid unnecessary computations when people array is empty.